question/classes/output/qbank_chooser_item.php

```diff
@@ -45,6 +47,8 @@ class qbank_chooser_item extends \core\output\chooser_item {
      * @param context $context The relevant context.
      */
     public function __construct($qtype, $context) {
+        debugging('Class qbank_chooser_item has been deprecated and moved to bank/editquestion,
```

Comment:
Can you please comment this and add to a TODO in tracker: https://tracker.moodle.org/browse/MDL-71679

Right now I'm getting a warning when editing a mod_quiz

```
Class qbank_chooser_item has been deprecated and moved to bank/editquestion, please use qbank_editquestion\qbank_chooser_item instead.
line 51 of /question/classes/output/qbank_chooser_item.php: call to debugging()
line 61 of /question/classes/output/qbank_chooser.php: call to core_question\output\qbank_chooser_item->__construct()
line ? of unknownfile: call to core_question\output\qbank_chooser->core_question\output\{closure}()
line 60 of /question/classes/output/qbank_chooser.php: call to array_map()
line 106 of /question/classes/output/qbank_chooser.php: call to core_question\output\qbank_chooser->__construct()
line 1072 of /mod/quiz/classes/output/edit_renderer.php: call to core_question\output\qbank_chooser::get()
line 126 of /mod/quiz/classes/output/edit_renderer.php: call to mod_quiz\output\edit_renderer->question_chooser()
line 211 of /mod/quiz/edit.php: call to mod_quiz\output\edit_renderer->edit_page()
```